SETUP.md

```diff
@@ -86,37 +85,43 @@ Additionally, if you want to test a particular version of spark, you may pass th
 
     python scripts/generate_conda_file.py --pyspark-version 2.4.0
 
-**NOTE** - for a PySpark environment, we need to set the environment variables `PYSPARK_PYTHON` and `PYSPARK_DRIVER_PYTHON` to point to the conda python executable.
-
-To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#macos-and-linux). Assuming that we have installed the environment in `/anaconda/envs/reco_pyspark`, we create the file `/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.sh` and add:
-
-```bash
-#!/bin/sh
-export PYSPARK_PYTHON=/anaconda/envs/reco_pyspark/bin/python
-export PYSPARK_DRIVER_PYTHON=/anaconda/envs/reco_pyspark/bin/python
-unset SPARK_HOME
-```
-
-This will export the variables every time we do `conda activate reco_pyspark`. To unset these variables when we deactivate the environment, we create the file `/anaconda/envs/reco_pyspark/etc/conda/deactivate.d/env_vars.sh` and add:
-
-```bash
-#!/bin/sh
-unset PYSPARK_PYTHON
-unset PYSPARK_DRIVER_PYTHON
-```
 </details>
 
 <details>
-<summary><strong><em>All environments</em></strong></summary>
+<summary><strong><em>PySpark & GPU environment</em></strong></summary>
 
-To install all three environments:
+To install the PySpark GPU environment:
```

Comment:
This is again confusing, it looks that we are doing Spark on GPU :-) 
Maybe we say in the title `PySpark & Python GPU environment` and then in the description below state clearly that the user can run PySpark **OR** Python GPU